src/Illuminate/Foundation/Http/Kernel.php

```diff
@@ -51,21 +51,21 @@ class Kernel implements KernelContract
     /**
      * The application's middleware stack.
      *
-     * @var array
+     * @var array<int, class-string|string>
```

Comment:
`string` is a supertype of `class-string`, so `class-string|string` can be shortened to `string`.

Reply:
@devfrey Done